.changeset/easy-spiders-wave.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": patch
+---
+
+Do a quick fix on build
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)